frontend/src/metabase/components/EditBar.jsx

```diff
@@ -30,6 +31,7 @@ class EditBar extends Component {
         )}
       >
         <div>
+          <Icon name="pencil" size={12} className="text-white mr1" />
```

Comment:
Just wanted to double-check: the `EditBar` is a shared component that's also used in dashboard and admin permissions edit modes. I see you removed the "You're editing ..." part from the dataset editor, but the "You're editing this dashboard" label is in place. Is that intended?

Reply:
Yep, I did know that, and did check how it looked on dashboards, but now you're making me question why I ended up not removing the "You're editing" text there on dashboards. Let me look at it again.

Reply:
I think it'll look funny 'cause the title of the dashboard is there below the edit bar:

![image](https://user-images.githubusercontent.com/2223916/146619405-9ee64958-b888-41a3-986c-302530e6f1ab.png)